portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsRichTextField.testcase

```diff
@@ -120,14 +120,46 @@ definition {
 	}
 
 	@description = "Verify that a help text can be added to a Rich Text Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133120 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Rich Text");
+
+		FormFields.editHelpText(
+			fieldHelpText = "Rich Text Help Text",
+			fieldName = "tip");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Rich Text",
+			fieldValue = "Rich Text Help Text");
+
+		CKEditor.addContentInWebContent(
+			content = "Test",
+			fieldLabel = "Rich Text");
```

Comment:
Instead of using a macro from CKEditor, can you please replicate this to a macro related to Forms? Thanks in advance.

Reply:
You can replicate in the same way the macro `viewRichTextValue` does in `FormFields.macro`.

Reply:
@mlan0  Changes done!